docs/proposals/ListView.md

```diff
@@ -0,0 +1,154 @@
+# Component Name
+
+The `ListPage` component will provide the structure and parts needed to display
+a list view.
+
+## Design Patterns
+
+There is a desire to provide developers with page templates to both speed up the
+development of pages but also provide our users with a more consistent
+experience.
```

Comment:
I wonder if Atlantis is the right place for this? I understand that we want this component and I see the value in it, but is there a risk in providing a ListPage layout to an open source project? Maybe there is a better place for this component?